Yearn-agents/InstantWithdrawl/src/utils.ts

```diff
@@ -0,0 +1,78 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+import { BigNumber } from "bignumber.js";
+
+export type UserDetails = {
+  account: string;
+  balance: string;
+};
+export type Mapping = {
+  [key: string]: Array<UserDetails>;
+};
+
+export const getAccounts = async (axios: any) => {
+  const query = `{
+    accountVaultPositions (orderBy:balancePosition, orderDirection:desc, first:150){
+      id, account{
+        id
+      },
+      balancePosition,
+      balanceShares,
+      vault{
+        id
+      }
+    }
+  }`;
```

Comment:
I think you shouldn't get the first `150`, you should do the query per vault and get the top `5`, `10` or something like that.